daemon/registry/credentialgraph.go

```diff
@@ -129,10 +118,10 @@ func (c *CredentialGraphClient) getGraph(ctx context.Context, query url.Values)
 	}
 
 	resp := []struct {
-		Keyring     *envelope.Signed  `json:"keyring"`
-		Members     json.RawMessage   `json:"members"`
-		Credentials []envelope.Signed `json:"credentials"`
-		Claims      []envelope.Signed `json:"claims"`
+		Keyring     *envelope.Signed              `json:"keyring"`
+		Members     json.RawMessage               `json:"members"`
+		Credentials []envelope.Signed             `json:"credentials"`
```

Comment:
yeah i'm just reusing `Signed` here as a generic container before finding the correct struct.